src/Pages/Login/Login.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from "react";
+import "./Login.scss";
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      pwpass: false,
+      errorActive: true,
+    };
+  }
+  loginEmail = (e) => {
+    this.setState({ email: e.target.value });
+    if (this.state.email.length > 5 && this.state.email.includes("@")) {
+      this.setState({ errorActive: true });
+    } else {
+      this.setState({ errorActive: false });
+    }
+  };
+
+  loginPassWord = (e) => {
+    this.setState({ password: e.target.value });
+    if (this.state.password.length >= 8) {
+      this.setState({ pwpass: true });
+    } else {
+      this.setState({ pwpass: false });
+    }
+  };
+
+  handleClick = () => {
+    fetch("http://10.58.4.149:8000/user/login", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => console.log(res.message));
+  };
+
+  render() {
+    console.log(this.state.errorActive);
+    return (
+      <main className="Login">
+        <div className="headerImg">
+          <span className="headeText">
+            <a href="#">Terms & Conditions</a>
+          </span>
+        </div>
+        <div className="loginBox">
+          <div className="contentHeader">
+            <h2 className="contentText">Welcome back!</h2>
+            <span className="contentSubText">
+              Enter your log in details below.
+            </span>
+          </div>
+          <div className="formBox">
+            <span className="formBoxText">Fields marked* are mandatory.</span>
+
+            <input
+              onChange={this.loginEmail}
+              className={this.state.errorActive ? "inputEmail" : "inputBox"}
+              type="text"
+              placeholder="Enter your email*"
+            />
+            <img
+              className={this.state.errorActive ? "succesImg" : "errorImg"}
+              alt="succes"
+              src="https://www.t2tea.com/on/demandware.static/Sites-UNI-T2-APAC-Site/-/en_AU/v1598112872308/images/validation_success.svg"
+            />
+            <span
+              className={this.state.errorActive ? "displayNone" : "displayOn"}
+            >
+              <img
+                className={
+                  this.state.errorActive ? "errorIconNone" : "errorIcon"
+                }
+                alt="succes"
+                src="https://www.t2tea.com/on/demandware.static/Sites-UNI-T2-APAC-Site/-/en_AU/v1598112872308/images/error_cross.svg"
+              />
+              {this.state.email.length === 0
```

Comment:
```suggestion
              {!email.length
```
0은 그 자체로 false값으로 취급 됩니다!